scss/civicrm/contact/pages/_events.scss

```diff
@@ -1,3 +1,15 @@
+.page-civicrm-event-search {
+  [class*=CRM_Event_Form_Task_] {
+    .crm-form-block > .crm-submit-buttons > .crm-button:first-of-type {
+      margin-left: 0;
```

Comment:
where did the margin come in the first place?

Reply:
The left margin comes from core civicrm CSS. On other pages, such as the form for adding a new activity, the left margin has been removed in a similar way.

Reply:
then why aren't we doing this in a more generic way, instead of doing it on a page-by page basis?

Reply:
To be honest, I have absolutely no idea why that was done in this way. It's my first work on these buttons, so will try to implement a more generic solution.

Reply:
There is a lot of this from @mukeshcompucorp first works on the theme in Feb - May of last year - perhaps he can clarify?

Reply:
@mihaelmladenov any style that is supposed to be generic cannot be redefined over and over for each page. if it was done like that (and it's very likely it is the case) then the style needs to be refactored so that the style is applied only once and we don't have to go back on it for every page) @mukeshcompucorp please provide assistance if necessary

Reply:
@mihaelmladenov @jamienovick this component is already made generic at https://github.com/civicrm/org.civicrm.shoreditch/blob/master/scss/civicrm/common/_buttons.scss#L108

Though margin does not get applied on all the pages so, other pages styled are fine without adding margin-left: 0. but, anything related to that style should be written in this file. so, it can prevent margin on other such pges as well.